adapters/seedingAlliance/seedingAlliance.go

```diff
@@ -0,0 +1,142 @@
+package seedingAlliance
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type SeedingAllianceAdapter struct {
+	endpoint string
+}
+
+func Builder(_ openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	bidder := &SeedingAllianceAdapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+func (a *SeedingAllianceAdapter) MakeRequests(request *openrtb2.BidRequest, extraRequestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	for i := range request.Imp {
+		if err := addTagID(&request.Imp[i]); err != nil {
+			return nil, []error{err}
+		}
+	}
+
+	request.Cur = append(request.Cur, "EUR")
```

Comment:
Need to confirm with a team if this is a correct way to set currency like this

Reply:
Discussed with team. This is ok from our perspective.  For bidder there are can be two "EUR". Do you want to check this array has it already before append?

Reply:
Thanks for confirming it!
That's a good point - just pushed a commit with the check.